src/styles/theme.ts

```diff
@@ -0,0 +1,46 @@
+import { ThemeOptions, createTheme } from "@mui/material/styles"
```

Comment:
Add custom font this way (see the example below):

`import { createTheme } from "@mui/material/styles"

declare module "@mui/material/styles" {
  interface Theme {
    otherFonts: {
      fontFamilySerif: string
    }
  }
  // allow configuration using `createTheme`
  interface ThemeOptions {
    otherFonts: {
      fontFamilySerif: string
    }
  }
}

export const theme = createTheme({
  palette: {
    mode: "light",
    primary: {
      main: "#311d13",
    },
    secondary: {
      main: "#99592a",
    },
    background: {
      default: "#ffffff",
      paper: "#fffaf6",
    },
  },
  otherFonts: {
    fontFamilySerif: "'Roboto', serif",
  },
  typography: {
    fontFamily: "'Source Sans 3', sans-serif",
    allVariants: {
      fontFamily: "'Source Sans 3', sans-serif",
    },
    body1: { fontFamily: "'Source Sans 3', sans-serif" },
    button: {
      fontWeight: 500,
      // textTransform: "capitalize",
    },
  },
  shape: {
    borderRadius: 12,
  },
})
`

**Remember to download all the fonts inside the html file or add them to the package json**